services/user.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth.hashers import make_password
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=make_password(password),
+        email=email if email else "",
+        first_name=first_name if first_name else "",
+        last_name=last_name if last_name else ""
+    )
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        return User.objects.get(id=user_id)
+    except User.DoesNotExist:
+        print(f"User with user_id {user_id} does not exist")
```

Comment:
Instead of printing an error message when a user does not exist, consider using logging or returning `None` to handle this case more gracefully.